npm-debug.log

```diff
@@ -0,0 +1,23 @@
+0 info it worked if it ends with ok
```

Comment:
Can you add a .gitignore entry for this?

Reply:
Done